tests/src/CMakeLists.txt

```diff
@@ -13,8 +13,5 @@ endif()
 if (TARGET gcl_gpu)
     add_library(mpi_gtest_main_gpu STATIC EXCLUDE_FROM_ALL mpi_test_driver.cpp)
     target_compile_definitions(mpi_gtest_main_gpu PRIVATE GT_HAS_CUDA)
-    target_link_libraries(mpi_gtest_main_gpu PUBLIC gridtools gtest MPI::MPI_CXX ${GT_CUDA_LIBRARIES})
-    if (GT_CUDA_INCLUDE_DIRS)
-        target_include_directories(mpi_gtest_main_gpu PRIVATE ${GT_CUDA_INCLUDE_DIRS})
-    endif()
+    target_link_libraries(mpi_gtest_main_gpu PUBLIC gridtools gtest MPI::MPI_CXX gcl_gpu)
```

Comment:
should we also link `mpi_gtest_main_cpu` with `gcl_cpu` ?

Reply:
Actually linking to gcl_gpu was a mistake as it then tries to compile the file as a cuda file...